src/toil/batchSystems/singleMachine.py

```diff
@@ -225,6 +226,13 @@ def getRescueBatchJobFrequency(cls):
         """
         return 5400
 
+    @staticmethod
+    def supportsWorkerCleanup():
+        """
+        Indicates whether the batch system supports the cleanup of workers on shutdown
```

Comment:
No need to duplicate the overridden method's docstring (unless you promise to keep these copies consistent from now until all eternity).